.npmrc

```diff
@@ -1 +1,3 @@
 shell-emulator=true
+# Required not to fail on Vercel
+package-manager-strict=false
```

Comment:
Don’t love that we need to add this, but I guess given it’s needed for smoke tests in the main repo, it’s ok 👍

Reply:
There was an update to this in 9.2.0 I think, let's try to comment it